tests/test.sh

```diff
@@ -121,6 +121,22 @@ fi
 
 check_valgrind_log "bad words"
 
+# Tests good words' root
+if test -f $TESTDIR/$NAME.root; then
+    hunspell $* -d $TESTDIR/$NAME <$TESTDIR/$NAME.good | grep -a '^+ ' | \
+        sed 's/^+ //' >$TEMPDIR/$NAME.root
+    if ! cmp $TEMPDIR/$NAME.root $TESTDIR/$NAME.root >/dev/null; then
+        echo "============================================="
+        echo "Fail in $NAME.root. Bad root?"
+        diff $TESTDIR/$NAME.root $TEMPDIR/$NAME.root
```

Comment:
I copied the line from other (bad words, morph, suggest) tests. Do you want other tests to be modified also?